soak_bench_scripts/soak.sh

```diff
@@ -3,12 +3,14 @@
 # Cargo operations, including removing artifcats from previous builds and building the binary needed for the soak testing
 # Git pull to ensure code is up to date
 # Output to log.log
-LOGFILE="/home/maidsafe/Projects/parsec/log.log"
+# Generate stack trace
+LOGFILE="/home/maidsafe/Projects/soak/parsec/log.log"
+export RUST_BACKTRACE=1
 
-cargo clean | tee $LOGFILE
-git pull | tee -a $LOGFILE
-cargo update | tee -a $LOGFILE
-cargo test --release --features=testing | tee -a $LOGFILE
+cargo clean |& tee -a $LOGFILE
```

Comment:
Do we really want to keep appending to the previous logfile now?

Reply:
I'm open to suggestions - do you think it would be better to create a new logfile each time the script is run (that was the case previously but I changed to append thinking to keep as much logging as poss)?

Reply:
I was just concerned it was a copy-paste error.  Ultimately, if the logfile grows too big, you'll need to delete it manually.  But if you're planning to keep the file for multiple runs like this, it might be worthwhile adding a line like `git log -1 --oneline` maybe after the `git pull` (pull would show what commit it got updated to if there were any changes, but it's not as obvious), and again for convenience at the end of the file?  Not required of course - up to you.

Reply:
OK, thanks - again no harm done including this in the script if it will potentially add some value.

Reply:
How about adding a line near the start of the script which moves any existing log file to logfile-{timestamp].bk?
Saves the live logfile from growing massive

Reply:
Good shout - I do something similar in the benchmark script.